packages/react/src/ActionGroup/ActionGroup.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export type ActionGroupProps = PropsWithChildren<HTMLAttributes<HTMLDivElement>>
+
+export const ActionGroup = forwardRef(
+  ({ children, className, ...restProps }: ActionGroupProps, ref: ForwardedRef<HTMLDivElement>) => (
+    <div {...restProps} ref={ref} className={clsx('ams-action-group', className)}>
```

Comment:
NLDS suggest to use a role attribute: https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-button-group--docs

Reply:
Cool, I’ve added it.